### PR TITLE
WRN-3542: Clean up "data-react-checksum" related code

### PR DESCRIPTION
**Pull request by hong6316**
### Issue Resolved / Feature Added
Clean up "data-react-checksum" related code

### Resolution
Removed react checksum related code

### Links
WRN-3542

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# unreleased
+
+* `PrerenderPlugin`:
```
Comment (MikyungKim) [inline CHANGELOG.md:3]: I think it's okay not to have a changelog for this change.

File: plugins/PrerenderPlugin/templates.js [hunk 2]
```diff
@@ -92,7 +92,6 @@ const multiLocale = (mapping) => mapping && `
 	var conf = details[lang] || details[lang.substring(0, 2)];
```
Comment (MikyungKim) [inline plugins/PrerenderPlugin/templates.js:92]: Please remove `and checksum` at line 89.

File: CHANGELOG.md (hunk 1 above)
Comment (hong6316) [reply CHANGELOG.md]: ok

File: plugins/PrerenderPlugin/templates.js (hunk 2 above)
Comment (hong6316) [reply plugins/PrerenderPlugin/templates.js:92]: ok

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**